_docs/contribute/dev-process.md

```diff
@@ -0,0 +1,42 @@
+title: Overview of the FIO Development Process
+description: Overview of the Process for Contributing to FIO
+---
+
+# Overview of the FIO Community's Development Process
+The FIO Core team does NOT do scrum, sprint, or any strict develeopment process. That being said, a given development cycle works something like this:
+
+- Initiatives are discussed high up, and prioritized. 
+- From initiatives there is analysis and initial scoping (done by VP, and Ed and lead engineers)
+- From analysis an Epic and project plan is made.
+- Lead engineers handle each epic in jira. From the Epic, stories are assigned and executed (we dont do sub tasks on FIO, just stories).
+- Project status is tracked in a status board auto generated from contents of the each Epic.
+- Daily standups are performed.
+- It's up to the lead to decide whether to operate with a meeting based culture or asynchronous communications or whatever they wish. For example, some use asynchronous and try hard to do very few official meetings (just get on the phone and lets work)…but you get to do any process on your projects if you can get buy in on from Eric and your peers on the team.
+
+## FIO Team Vibe
+- The FIO team has a “positivity first” policy: Be productive, be positive, be helpful, be truthful.
+- The FIO team has a “safe space” policy: If you are stressed and need to blow up, do you!... but do it without demeaning others and always remain respectful to others.
+- The FIO team has a “no assholes allowed” policy: Enough said.
+
+## FIO Team Environment Information and Links
+
+|Description  |Links |
+|---|---|
+|FIO is an open community so you can see and learn any aspect of FIO. Check out the list of approved improvement proposals!| [FIO Improvement Proposals](https://github.com/fioprotocol/fips) |
+|The FIO developer hub (devhub) provides documentation assist the community with understanding FIO. |[FIO Devhub](https://developers.fioprotocol.io/docs/developers/)|
+|FIO uses smart contracts to implement all business logic. Smart contracts are implemented in C++.|[FIO Smart Contracts](https://github.com/fioprotocol/fio.contracts) |
+|FIO is a copy (or fork) of the EOSIO chain so you can use any dev docs for EOSIO to help you in learning about FIO.
+FIO has “core code”, the operating system upon which the smart contracts execute (called the FIO Core, and implemented in C++). |[FIO Core](https://github.com/fioprotocol/fio) |
+|FIO provides a developer runtime environment (which works on IOS ubuntu). A developer can “run” the blockchain locally for dev and testing.|[FIO Developer Environment](https://github.com/fioprotocol/fio.devtools) |
+|The FIO protocol has a block producer community that operates and maintains the chain worldwide. |[FIO Block Producers](https://developers.fioprotocol.io/docs/bp/) |
+|The FIO protocol uses worker proposals to scope jobs, and jobs are paid in crypto. |[FIO Worker Proposals](https://developers.fioprotocol.io/docs/contribute/onboarding) |
```

Comment:
Change link to: https://fioprotocol.atlassian.net/wiki/spaces/DAO/pages/64192796/How+to+Contribute+to+FIO+Protocol

Reply:
fixed